Tests/fixtures/tv.yml

```diff
@@ -6966,3 +6966,2506 @@
+  os:
+    name: GNU/Linux
+    version: ""
+    platform: ARM
+  client:
+    type: browser
+    name: Opera Devices
+    version: 4.9.0.176
+    engine: Blink
+    engine_version: ""
+  device:
+    type: tv
+    brand: Fuego
+    model: MB211
+  os_family: GNU/Linux
+  browser_family: Opera
+- 
+  user_agent: Mozilla/5.0 (Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.143 Safari/537.36 OPR/40.0.2207.0 OMI/4.9.0.176.DOM3.104 Model/Vestel-MB211 VSTVB MB200 HbbTV/1.2.1 (; TECHNIKA; MB211; 0.38.5.0; _TV_NT72563_2017;) SmartTvA/3.0.0
+  os:
+    name: GNU/Linux
+    version: ""
+    platform: ARM
+  client:
+    type: browser
+    name: Opera Devices
+    version: 4.9.0.176
+    engine: Blink
+    engine_version: ""
+  device:
+    type: tv
+    brand: Technika
+    model: MB211
+  os_family: GNU/Linux
+  browser_family: Opera
+- 
+  user_agent: Mozilla/5.0 (Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/44.0.2403.130 Safari/537.36 OPR/31.0.1890.0 OMI/4.6.1.40.Dominik2.193 VSTVB MB100 HbbTV/1.2.1 (; GOGEN; MB110; 2.32.15.0; ;) SmartTvA/3.0.0
+  os:
+    name: GNU/Linux
+    version: ""
+    platform: ARM
+  client:
+    type: browser
+    name: Opera Devices
+    version: 4.6.1.40
+    engine: Blink
+    engine_version: ""
+  device:
+    type: tv
+    brand: GoGEN
+    model: MB110
+  os_family: GNU/Linux
+  browser_family: Opera
+- 
+  user_agent: Mozilla/5.0 (Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/44.0.2403.130 Safari/537.36 OPR/31.0.1890.0 OMI/4.6.1.40.Dominik2.193 VSTVB MB100 HbbTV/1.2.1 (; KUBO; MB120; 2.40.14.0; ;) SmartTvA/3.0.0
+  os:
+    name: GNU/Linux
+    version: ""
+    platform: ARM
+  client:
+    type: browser
+    name: Opera Devices
+    version: 4.6.1.40
+    engine: Blink
+    engine_version: ""
+  device:
+    type: tv
+    brand: KUBO
+    model: MB120
+  os_family: GNU/Linux
+  browser_family: Opera
+- 
+  user_agent: Mozilla/5.0 (Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.143 Safari/537.36 OPR/40.0.2207.0 OMI/4.9.0.176.DOM3.169 Model/Vestel-MB130 VSTVB MB100 HbbTV/1.2.1 (; VOX; MB130; 3.47.10.0; _TV_G10_2017;) SmartTvA/3.0.0
+  os:
+    name: GNU/Linux
+    version: ""
+    platform: ARM
+  client:
+    type: browser
+    name: Opera Devices
+    version: 4.9.0.176
+    engine: Blink
+    engine_version: ""
+  device:
+    type: tv
+    brand: VOX
+    model: MB130
+  os_family: GNU/Linux
+  browser_family: Opera
+- 
+  user_agent: Mozilla/5.0 (Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/44.0.2403.130 Safari/537.36 OPR/31.0.1890.0 OMI/4.6.1.40.Dominik2.193 VSTVB MB100 HbbTV/1.2.1 (; ELECTRONIA; MB110; 2.32.0.0; ;) SmartTvA/3.0.0
+  os:
+    name: GNU/Linux
+    version: ""
+    platform: ARM
+  client:
+    type: browser
+    name: Opera Devices
+    version: 4.6.1.40
+    engine: Blink
+    engine_version: ""
+  device:
+    type: tv
+    brand: Electronia
+    model: MB110
```

Comment:
ELECTRONIA brand may exist

often `MB[0-9]{3,4}` are not models, it is a common MainBoard/firmware. Moreover, these symbols are found for different brands 

http://www.zeppy.io/discover/gb/mb110

Reply:
Should I also correct some regexes that start with LF as model name? As far as I can see is a firmware version, and not a model. Right now are seen as models.
`Opera/9.80 (Linux mips; U; NETRANGEMMH;HbbTV/1.1.1;CE-HTML/1.0;TCL LF1V048; en) Presto/2.10.287 Version/12.00
Opera/9.80 (Linux armv7l; U; NETRANGEMMH;HbbTV/1.1.1;CE-HTML/1.0;Vendor/THOMSON;SW-Version/V8-MT51F01-LF1V466;Cnt/FRA;Lan/eng) Presto/2.12.362 Version/12.11`

Reply:
> Should I also correct some regexes that start with LF as model name? As far as I can see is a firmware version, and not a model. Right now are seen as models.
> `Opera/9.80 (Linux mips; U; NETRANGEMMH;HbbTV/1.1.1;CE-HTML/1.0;TCL LF1V048; en) Presto/2.10.287 Version/12.00`
> `Opera/9.80 (Linux armv7l; U; NETRANGEMMH;HbbTV/1.1.1;CE-HTML/1.0;Vendor/THOMSON;SW-Version/V8-MT51F01-LF1V466;Cnt/FRA;Lan/eng) Presto/2.12.362 Version/12.11`

yes replace

Reply:
https://www.hbbtv.org/resource-library/specifications/
I have noticed that if the habtv does not meet the specification, then it is the motherboard or firmware.
Often such useragents contain the DOM2-3 prefixes

example 

`Mozilla/5.0 (Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.143 Safari/537.36 OPR/40.0.2207.0 OMI/4.9.0.237.DOM3.223 Model/Vestel-MB211 VSTVB MB200 HbbTV/1.2.1 (; GOGEN; MB211; 0.59.2.2; _TV_NT72563_2017;) SmartTvA/3.0.0
`  prefix  `DOM3`